CONTRIBUTING.md

```diff
@@ -208,13 +208,12 @@ during development.
   or role that you use for Github actions.
 
 _Note_: All integration tests are set up to run in `eu-west-*` region.  If you prefer
-to run in a different region you must update the region in each test before running it
-
+to run in a different region you must update the region in each test before running it.
 
 #### run all tests
 
 ```bash
-$ AWS_PROFILE=<profile> AWS_DEFAULT_REGION=<region> poetry run behave integration-tests/features --junit --junit-directory build/behave
+$ AWS_DEFAULT_REGION=eu-west-1 poetry run behave integration-tests/features --junit --junit-directory build/behave
```

Comment:
i don't think we want to make it look like you need to set it to only one region because it's possible to run tests in a regions other than `eu-west-1`.

Reply:
You kind of do need to run it in `eu-west` though:
```
% grep -r eu-west integration-tests 
integration-tests/features/stack-output-resolver.feature:    Given stack "6/2/A" does not exist in "eu-west-1"
integration-tests/features/stack-output-resolver.feature:    and stack "6/2/B" does not exist in "eu-west-2"
integration-tests/features/stack-output-resolver.feature:    and stack "6/2/C" does not exist in "eu-west-3"
integration-tests/features/stack-output-resolver.feature:    Then stack "6/2/A" in "eu-west-1" exists in "CREATE_COMPLETE" state
integration-tests/features/stack-output-resolver.feature:    and stack "6/2/B" in "eu-west-2" exists in "CREATE_COMPLETE" state
integration-tests/features/stack-output-resolver.feature:    and stack "6/2/C" in "eu-west-3" exists in "CREATE_COMPLETE" state
integration-tests/sceptre-project/config/config.yaml:region: eu-west-1
integration-tests/sceptre-project/config/6/2/A.yaml:region: eu-west-1
integration-tests/sceptre-project/config/6/2/B.yaml:region: eu-west-2
integration-tests/sceptre-project/config/6/2/C.yaml:region: eu-west-3
integration-tests/sceptre-project/config/external-stack-output/outputter.yaml:region: eu-west-1
```

Are you suggesting I change it to mention you could also set it to another region if you edit these files first?

Reply:
Anyway, have another look. I've made some changes.

Reply:
> Are you suggesting I change it to mention you could also set it to another region if you edit these files first?

yes, that is correct